cloudflare.go

```diff
@@ -67,16 +67,17 @@ func (api *API) SetAuthType(authType int) {
 
 // ZoneIDByName retrieves a zone's ID from the name.
 func (api *API) ZoneIDByName(zoneName string) (string, error) {
-	res, err := api.ListZones(zoneName)
+	//TODO: paginate
```

Comment:
Remove this comment, as not necessary. The FilterZones call should search for the exact name via the `name=${name}` param.